### DISCUSSION TITLE
Is it possible to specify the status file path ?

### DISCUSSION THREAD

**Discussion by qingant** [ClickHouse/ClickHouse#29606] [General]
We are running Clickhouse on top of some shared file system, we want to start multi Clickhouse in different servers on same directory to enable smooth failover. Well, it's prevented by status file lock ? it there any method that we can config the status file path ?

**Comment by den-crane** [comment]
You cannot use shared file system. Clickhouse does not support this. You will corrupt the database if two instances of Clickhouse start to use the same datafiles.
